src/manager.cpp

```diff
@@ -30,6 +30,13 @@ sdbusplus::async::task<> Manager::init()
     co_await sdbusplus::async::execution::when_all(
         parseConfiguration(), _extDataIfaces->startExtDataFetches());
 
+    // TODO: Explore the possibility of running FullSync and Background Sync
+    // concurrently
+    if (_extDataIfaces->bmcRedundancy())
+    {
+        co_await startFullSync();
```

Comment:
This is just so if the service is restarted it can do a full sync to sync anything it missed?  because normally anytime this starts redundancy won't be enabled yet.